molgenis-api-data/src/main/java/org/molgenis/api/data/v2/CopyEntityRequestV2.java

```diff
@@ -12,6 +12,7 @@
   @NotNull
   public abstract String getNewEntityName();
 
+  @SuppressWarnings("unused")
```

Comment:
Both IntelliJ and Sonarlint are not reporting issues in my IDE?

Similar to the AttributeResponse comments I think that we can remove all create methods and builders from AutoValue objects unless they are used in unit tests (which they should).

Reply:
As discussed, figuring out which methods can be removed or should be suppressed is work for another time. For now I'll remove the annotations